build.gradle

```diff
@@ -212,7 +212,8 @@ sonarqube {
     property "sonar.pitest.reportsDirectory", "build/reports/pitest"
     property "sonar.exclusions", "src/main/java/uk/gov/hmcts/reform/sscs/HearingsApiApplication.java," +
             "src/main/java/uk/gov/hmcts/reform/sscs/config/**," +
-            "src/main/java/uk/gov/hmcts/reform/sscs/model/**,"
+            "src/main/java/uk/gov/hmcts/reform/sscs/model/**," +
+            "src/main/java/uk/gov/hmcts/reform/sscs/service/ReferenceDataServiceHolder.java,"
```

Comment:
You don't need to suppress, found out how to test scenarios like this:
https://github.com/hmcts/sscs-hearings-api/commit/de4ef4b193c01b0e11679237a0c56bcf822bf863#diff-d31f9e3efa7fefd1667c39835b2a8f69d25648cbfaf4f9a786f76ceab47885b0

Reply:
Ok. I removed this line.